packages/codemods/src/codeMods/types.ts

```diff
@@ -1,28 +1,30 @@
 import { SourceFile } from 'ts-morph';
 
-export interface ICodeModResult {
+// tslint:disable-next-line: interface-name
```

Comment:
Instead of disabling per line you can modify the package's tslint.json
```
{
  "extends": ["@uifabric/tslint-rules"],
  "rules": {
    "interface-name": [true, "never-prefix"]
  }
}
```

Reply:
I'll plan do that once this has been checked in.

Reply:
why postpone to next checkin? it's more suitable to do it now/in this pr

Reply:
@joschect - we should fix this in this round